src/StatsBase.jl

```diff
@@ -242,6 +242,14 @@ export
     CronbachAlpha,        # the type to represent Cronbach's alpha scores
     cronbachalpha         # function to compute Cronbach's alpha scores
 
+# https://github.com/JuliaLang/julia/pull/30630
+if VERSION < v"1.2.0-DEV.125" # 1da48c2e4028c1514ed45688be727efbef1db884
+    require_one_based_indexing(A...) = !Base.has_offset_axes(A...) || throw(ArgumentError(
+        "offset arrays are not supported but got an array with index other than 1"))
+else
+    using Base: require_one_based_indexing
+end
+
```

Comment:
I prepare a PR that updates the Julia compat to 1.10 but this can be done separately from this PR.

Reply:
Removed it again in https://github.com/JuliaStats/StatsBase.jl/pull/953/commits/dc23262892b000e6e7cec0a0ce7de373f5230617